src/cloud-sql-instance.ts

```diff
@@ -43,6 +43,13 @@ interface CloudSQLInstanceOptions {
   sqlAdminFetcher: Fetcher;
 }
 
+interface RefreshableValues {
+  ephemeralCert: SslCert;
+  host: string;
```

Comment:
This should be `ipAddress` I believe or even `ipAddresses` to store all the IP addresses right?

Reply:
oh this value actually gets stored post-parsing ([done here in L189](https://github.com/GoogleCloudPlatform/cloud-sql-nodejs-connector/pull/219/files#diff-0ccad9395bd6c6e5a399bd0f8869ded90a2b12ccf05bc5004dd3189e2fb42692R189)), so we store only the selected IP address under a property named `host` (which is the property name used to open the TLS socket in node APIs). 

When thinking about name-alignment there's some other factors to consider such that in JavaScript matching property names helps with using a cleaner destructuring assignment, so there's sometimes an incentive to stick with a given property name.

I'm def happy to make concessions for the sake of aligning the connectors but in this case I believe that switching to storing the unparsed `ipAddress` is a larger change then I'd be willing to make at this point.